### DISCUSSION TITLE
i need help

### DISCUSSION THREAD

**Discussion by quanq026** [ok-oldking/ok-wuthering-waves#97] [Q&A]
![image](https://github.com/user-attachments/assets/e1321a4d-ca06-4982-bd38-e9c6a55fc8c8)

**Comment by Annn2022** [comment]
![image](https://github.com/user-attachments/assets/be6bbc2f-f0aa-4f4e-ac00-3b1de510af1f)

**Reply by Annn2022** [reply]
in this window to use
